### DISCUSSION TITLE
Understanding underlying linear model equation

### DISCUSSION THREAD

**Discussion by dustinkincaid** [lindeloev/mcp#144] [Q&A]
Hi, I fitted my full, single changepoint model using the following:

```
  model_full = list(
    log_c ~ 1 + log_q,  # Intercept and slope in the first segment
    ~ 0 + log_q  # Joined slope in the second segment
  )
```

I'm trying to figure out the underlying linear model so that I can manually predict y values on the second segment (for manual plotting purposes down the road) and make sure I am reporting the slope for the second segment correctly. I thought the equation would be the following, but I don't think that's correct: yi = β01 + β1xi + β2xi

Any help would be appreciated!

**Comment by lindeloev** [comment]
In absolute numbers, the intercept of the second segment is the function value of the first segment at the change point (`β0 + min(cp_1, log_q) * β1`). The "0" in the formular for segment 2 indicates that there is no abrupt intercept change at the change point. From there, the expected value for `log_c` only changes as a slope on `log_q`. So I guess

    log_c = β11 + min(cp_1, log_q) * β12 + (log_q - cp_1) * β2

You can see [a writeup of this example (two joined slopes) on the mcp website](https://lindeloev.github.io/mcp/articles/formulas.html#modeling-slope-change-points).
